recipes-openxt/initrdscripts/initramfs-xenclient/init.sh

```diff
@@ -28,7 +28,14 @@ DEFINIT=/sbin/init
 FIRSTBOOT_FLAG=/boot/system/firstboot
 
 is_tpm_2_0 () {
-    [ -e /sys/class/tpm/tpm0/device/description ] && cat /sys/class/tpm/tpm0/device/description | grep "2.0" &>/dev/null
+   files=$(find /sys/ | grep "description$")
+   for file in $files; do
+      tpm=$(cat ${file})
+      if [ "${tpm}" == "TPM 2.0 Device" ]; then
```

Comment:
The proper syntax is a single = sign

Reply:
```==``` is a bash alias for ```=```, and it's behaviour only changes between double-brackets.

Reply:
Actually `[` is a binary, not a built-in, and `man [` doesn't mention `==`, most `[` implementations just happen to alias `==` to `=` because it's a common mistake.